src/main/webapp/app/exercises/programming/participate/programming-submission.service.ts

```diff
@@ -109,7 +109,7 @@ export class ProgrammingSubmissionService implements IProgrammingSubmissionServi
             return acc + buildingSubmissionsOfExercise;
         }, 0);
 
-        // For every 100 submissions, we increase the expected time by 1 minute.
+        // For every 100 submissions, we increase the expected time by 1 minute. // TODO this is wrong, should be 4 min?
```

Comment:
Is this something you are currently discussing and that will be part of the scope or something for the future? If it's the latter, please create an issue for it.

Reply:
I can do so, will link the issue here.

Reply:
See #4000 (I just hit the 4000 🚀 )